pkg/controller/machineset_test.go

```diff
@@ -485,8 +485,6 @@ var _ = Describe("machineset", func() {
 			Err := c.reconcileClusterMachineSet(Key)
 
 			waitForCacheSync(stop, c)
-			machines, _ = c.controlMachineClient.Machines(testNamespace).List(context.TODO(), metav1.ListOptions{})
-			//Expect(len(machines.Items)).To(Equal(int(testMachineSet.Spec.Replicas)))
```

Comment:
I am not sure whey `Expect` line was already commented here because according to what this testcase is for , this line is required. Did you check by uncommenting it if this testcase runs?

Reply:
Yes, the commented part was already commented before my changes. Ideally, it should create the machine but somehow it doesn't. May be it some issue with the mocking we do.